09. cli-and-http-request/clihelper/cliStart.go

```diff
@@ -0,0 +1,18 @@
+package clihelper
+
+type CmdRunner struct {
+	CliBindingProperties CliBindingProperties
+}
+
+type Writer interface {
+	Writer(wc WriterConfiguration, line string)
+}
+
+type StdInParameter struct {
```

Comment:
Let's move all the structs to `structs` and this one to `structs\StdInParamters.go` @Nafrose